app/src/main/res/layout/component_collection_creation.xml

```diff
@@ -181,7 +180,7 @@
             android:background="?android:attr/selectableItemBackground"
             android:text="@string/create_collection_save"
             android:textColor="?textActionPrimary"
-            app:fontFamily="@font/metropolis_medium"
+            android:textStyle="bold"
```

Comment:
The link also points to snackbar design. For all places where bold style was applied I did not add font weight because it would change the current text appearance.